cmd/notary/main.go

```diff
@@ -11,20 +10,25 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
+	"github.com/docker/docker/pkg/homedir"
 	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/version"
 )
 
-const configFileName string = "config"
-const defaultTrustDir string = ".notary/"
-const defaultServerURL = "https://notary-server:4443"
-const idSize = 64
+const (
+	configFileName   string = "config"
+	defaultTrustDir  string = ".notary/"
+	defaultServerURL        = "https://notary.docker.io"
```

Comment:
if i as a docker user want to use `notary` the cmd line tool to manage my
keys what server do i point it at, because I think we can all agree (at
least for the time being, the most popular use case for the notary cmd line
tool will be with docker.

On Mon, Aug 10, 2015 at 10:12 AM, Diogo Mónica <notifications@github.com>
wrote:

> In cmd/notary/main.go
> <https://github.com/docker/notary/pull/180#discussion_r36656769>:
>
> >  	"github.com/docker/notary/pkg/passphrase"
> >  	"github.com/docker/notary/version"
> >  )
> >
> > -const configFileName string = "config"
> > -const defaultTrustDir string = ".notary/"
> > -const defaultServerURL = "https://notary-server:4443"
> > -const idSize = 64
> > +const (
> > +	configFileName   string = "config"
> > +	defaultTrustDir  string = ".notary/"
> > +	defaultServerURL        = "https://notary.docker.io"
>
> Note that there is no current public notary server. We would have to deal
> with GUN ownership for that to be a reality.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/notary/pull/180/files#r36656769>.
>